library/ZendService/Twitter/Twitter.php

```diff
@@ -1355,4 +1399,24 @@ protected function friendsIds($id, array $params = [])
         $response = $this->get($path, $params);
         return new Response($response);
     }
+
+
+    /**
+     * Returns the subscribers of the specified list. Private list subscribers
+     * will only be shown if the authenticated user owns the specified list.
+     *
+     * Returns the next cursor if there are more to be returned.
+     *
+     * @param  int|string $id
+     * @param  array $params
+     * @return Response
+     */
+    protected function listsSubscribers($id, array $params = [])
```

Comment:
`private`, unless there are strong reasons to make it `protected`. If there are, then that needs to be documented in the docblock too